tests/multipleBackend/objectPut.js

```diff
@@ -0,0 +1,148 @@
+import assert from 'assert';
+
+import { cleanup, DummyRequestLogger, makeAuthInfo } from '../unit/helpers';
+import { ds } from '../../lib/data/in_memory/backend';
+import bucketPut from '../../lib/api/bucketPut';
+import objectPut from '../../lib/api/objectPut';
+import DummyRequest from '../unit/DummyRequest';
+
+const log = new DummyRequestLogger();
+const canonicalID = 'accessKey1';
+const authInfo = makeAuthInfo(canonicalID);
+const namespace = 'default';
+const bucketName = 'bucketname';
+const postBody = Buffer.from('I am a body', 'utf8');
```

Comment:
I would just call this 'body' since it's the data being put on a put request rather than a post request.